precompiles/authorization/events.go

```diff
@@ -43,50 +43,6 @@ type EventRevocation struct {
 	TypeUrls []string
 }
 
-// EmitAllowanceChangeEvent creates a new allowance change event emitted on IncreaseAllowance
-// and DecreaseAllowance transactions.
-func EmitAllowanceChangeEvent(args cmn.EmitEventArgs) error {
```

Comment:
How come we are removing the `EmitAllowanceChangeEvent` method but the `AllowanceChangeEvent` is still there? 🤔 
https://github.com/evmos/evmos/blob/90bb20e32af788df9f8fee18fdf8253286004e6d/precompiles/authorization/events.go#L31C1-L37

Reply:
`AllowanceChangeEvent` is still used in the `staking` extension authorization flow for Increase / Decrease allowance. This will also be refactored in a follow up PR to just emit an `Approval` event with the new allowance!